src/main/java/bio/terra/service/search/SearchService.java

```diff
@@ -46,12 +44,14 @@
     private final BigQueryPdao bigQueryPdao;
     private final RestHighLevelClient client;
 
+    @Value("${elasticsearch.numShards}")
```

Comment:
This seems like a good use of `@Value`. Although, maybe it should be in `ElasticSearchConfiguration`? I don't have a strong opinion either way.